backend/src/controllers/cars/cars.controller.ts

```diff
@@ -66,4 +66,23 @@ const getUsedOptions = async (
   }
 };
 
-export { carsSearchAutoria, getBrands, getModelsOfBrand, getUsedOptions };
+const getCitiesOfRegion = async (
+  req: TypedRequestParams<{ id: string }>,
+  res: Response,
+  next: NextFunction,
+): Promise<void> => {
+  try {
+    const models = await carsService.getCities(req.params.id);
```

Comment:
Cities are not part of the `car`'s ecosystem. They should be handled in a separate service

Reply:
Oh-oh. Then it somehow slipped through my attention. Could you please create a separate service for the locations?